stories/FormControl.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { FormControl, InputField, Help } from '../src'
+
+import markdown from './info/atoms/formControl.md'
+
+storiesOf('FormControl', module)
+    .addParameters({
+        notes: {
+            markdown,
+        },
+    })
+    .add('Default', () => (
```

Comment:
I think this story only makes sense when using 2 `FormControl`s

Reply:
True. I considered a wrapper with a border, but that would create as much confusion as it would clarify things. Two FormControls makes sense. Added them just now.